config/initializers/active_job_uniqueness.rb

```diff
@@ -1,35 +1,37 @@
 # frozen_string_literal: true
 
-ActiveJob::Uniqueness.configure do |config|
-  # Global default expiration for lock keys. Each job can define its own ttl via :lock_ttl option.
-  # Stategy :until_and_while_executing also accept :on_runtime_ttl option.
-  config.lock_ttl = 1.day
+Rails.application.reloader.to_prepare do
```

Comment:
This `to_prepare` line is the only substantial change in this file, per https://guides.rubyonrails.org/upgrading_ruby_on_rails.html#autoloading-during-initialization. The rest of the lines are just indentation shifts.